rdrf/useraudit/test_urls.py

```diff
@@ -0,0 +1,23 @@
+from django.contrib import admin
+from .views import test_request_available
+
+try:
+    from django.urls import include, path
+    PRE_DJANGO_2 = False
+except ImportError:
+    from django.conf.urls import include, url
+    PRE_DJANGO_2 = True
+
+
+admin.autodiscover()
+
+if PRE_DJANGO_2:
```

Comment:
Do we need this `PRE_DJANGO_2` logic?

Reply:
Nope, removed those files & middleware + logic related to django 1 support